src/statistics.jl

```diff
@@ -327,7 +327,7 @@ function apply_statistic(stat::HistogramStatistic,
             x_min = Gadfly.concrete_minimum(values)
             span = x_max - x_min
             binwidth = span / d
-            bincounts ./= sum(bincounts) * binwidth
+            bincounts = bincounts ./ sum(bincounts) * binwidth
```

Comment:
No, I mean do `binmean = bincounts ./ sum(bincounts) * binwidth`.   Then `bincounts` is always an array of integers, and `binmean` is always an array of floats.   In subsequent lines, work with `binmean`.

(I realize that it is probably not performance critical to be type-stable here, but it is better style in Julia to retain type stability if it is not too hard.)